app/scripts/modules/core/src/pipeline/status/ExecutionStatus.tsx

```diff
@@ -112,8 +112,8 @@ export class ExecutionStatus extends React.Component<IExecutionStatusProps, IExe
       <div className="execution-status-section">
         <span className={`trigger-type ${this.state.sortFilter.groupBy !== name ? 'subheading' : ''}`}>
           <h5 className="build-number"><ExecutionBuildLink execution={execution}/></h5>
-          <h5 className="execution-type">
-            {execution.trigger.dryRun && '[DRY RUN] '}
+          <h5 className={`execution-type ${execution.trigger.dryRun && 'execution-dry-run'}`}>
```

Comment:
Switch this to a ternary, otherwise the generated class can be something unexpected such as:
`execution-type false` or `execution-type null`